src/Sulu/Component/Content/Document/Structure/PropertyValue.php

```diff
@@ -87,7 +87,7 @@ public function offsetSet($offset, $value)
             return;
         }
 
-        $this->value[$offset] = array_merge($this->value[$offset], $value);
+        $this->value[$offset] = $value;
```

Comment:
@nickvanderzwet I'm seeing this as errors in the monitoring of our project, do you know if the client is also experiencing problems as a result?